README.md

```diff
@@ -10,30 +10,51 @@ Expected environment:
   - Node Feature Discovery (NFD) Operator
   - Kernel Module Management (KMM) Operator
 
-### SFC MachineConfig
+### SFC out of tree driver
 
-To apply the SFC MachineConfig, it is a prerequisite that there exists a local docker registry containing an image with the SFC module built and present in /opt in the node filesystem. This is expected to be present at the following location:
+The recommended way of installing onload's sfc driver is using [KMM](https://github.com/rh-ecosystem-edge/kernel-module-management/)
 
+#### Deploy using KMM
+
+Before you apply the `Module` custom resource for the sfc you must remove the
+existing in-tree driver.
+
+For each appropriate node in the cluster:
+```console
+# rmmod sfc
 ```
-image-registry.openshift-image-registry.svc:5000/openshift-kmm/sfc-module
+
+Then apply the manifest:
+
+```console
+$ oc apply -f sfc/kmm/sfc-module.yaml
 ```
 
-To generate this image, please use SFC module:
+#### Day-0 deployment using MachineConfig
+
+To apply the SFC MachineConfig, it is a prerequisite that there exists a local
+docker registry containing an image with the SFC module built and present in
+`/opt` in the node filesystem. This is expected to be present at the following
+location:
+
 ```
-$ oc apply -f sfc/kmm/sfc-module.yaml
+image-registry.openshift-image-registry.svc:5000/openshift-kmm/sfc-module
 ```
 
-The generated image must be labelled with the exact kernel version necessary for the worker nodes. All worker nodes must share the same kernel version.
+The generated image must be labelled with the exact kernel version necessary
+for the worker nodes. All worker nodes must share the same kernel version.
 
 Apply the SFC MachineConfig:
 
-```
+```console
 $ butane sfc/mco/99-sfc-machineconfig.bu -d sfc/mco/ -o sfc/mco/99-sfc-machineconfig.yaml
 $ oc apply -f sfc/mco/99-sfc-machineconfig.yaml
 ```
 
-Please ensure that the openshift version specified within the above butane file tracks the appropriate specification revision.
-https://github.com/coreos/butane/blob/main/docs/specs.md#stable-specification-versions
+Please ensure that the openshift version specified within the above butane file
```

Comment:
```suggestion
Please ensure that the OpenShift version specified within the above butane file
```

And perhaps worth applying the same changes for the "Disable chronyd" section, or de-duplicate it?